connector_prestashop_environment/models/prestashop_backend.py

```diff
@@ -0,0 +1,25 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class PrestashopBackend(models.Model):
+    _name = "prestashop.backend"
+    _inherit = ["prestashop.backend", "server.env.mixin"]
+
+    @property
+    def _server_env_fields(self):
+        base_fields = super()._server_env_fields
+        mail_fields = {
```

Comment:
Just curious, why is the name `mail_fields`?

Reply:
Bad copy paste I guess ;)